src/zope/testbrowser/cookies.py

```diff
@@ -50,11 +50,24 @@ def info(self):
         return self.message
 
 
+DAY_OF_WEEK = ["Mon", "Tue", "Wed", "Thu", "Fri", "Sat", "Sun"]
```

Comment:
I usually make constants tuples so they're immutable. That's probably just me and it probably doesn't matter.